app/assets/stylesheets/application.scss

```diff
@@ -25,24 +25,19 @@ $govuk-page-width: 1200px;
 $light-blue: tint($department-for-education-websafe, 90%);
 
 @import 'dfe-header';
-@import 'hfeyp-v2';
 
 @import 'video';
 @import 'page-contents';
-@import '_metadata';
+@import 'metadata';
 
-@import 'dfe/print';
-@import 'dfe/about';
-@import 'dfe/card';
-@import 'dfe/side-nav';
-
-@import '_subnav';
-@import '_eyfs-header';
-@import '_eyfs-mobile-navigation';
-@import '_cookie-banner';
-@import '_links';
-@import '_flash';
+@import 'print';
+@import 'card';
+@import 'side-nav';
+@import 'other-resources';
+@import 'cta';
 
+@import 'subnav';
```

Comment:
This file has styling (that isn't imported) which can still be tidied up and removed. `app-sidebar` for example.

Reply:
I have removed subnav in the next commit